editor/connections_dialog.cpp

```diff
@@ -872,19 +872,18 @@ void ConnectionsDock::_make_or_edit_connection() {
 	Ref<Script> scr = target->get_script();
 	if (!scr.is_null() && !ClassDB::has_method(target->get_class(), cd.method)) {
 		// There is a chance that the method is inherited from another script.
-		bool found_inherited_function = false;
-		Ref<Script> inherited_scr = scr->get_base_script();
-		while (!inherited_scr.is_null()) {
-			int line = inherited_scr->get_language()->find_function(cd.method, inherited_scr->get_source_code());
-			if (line != -1) {
-				found_inherited_function = true;
+		bool found_member_function = false;
+		Ref<Script> top_scr = scr;
```

Comment:
I'd argue that `top_scr` is a bit misleading, `inherited_scr` was more correct and informative imo